desktop/apps/consignments2/client/actions.js

```diff
@@ -201,8 +201,9 @@ export function handleImageUpload (file) {
         progress: (percent) => {
           // console.log("<3 progress bars, file is this % uploaded: ", percent)
         },
-        done: (src) => {
-          dispatch(uploadImageToConvection(src, file.name))
+        done: (src, geminiKey, bucket) => {
+          const key = `${geminiKey}/${file.name}`
+          dispatch(uploadImageToGemini(key, bucket, file.name))
         }
       }
       await gemup(file, options)
```

Comment:
FWIW gemup doesn't return a promise, so `await` isn't doing anything here (I definitely accept PRs for that though 😁 ). It's not affecting anything though b/c you're using the `done` callback.